templates/grpc.gemspec.template

```diff
@@ -34,7 +34,7 @@
     s.require_paths = %w( src/ruby/lib src/ruby/bin src/ruby/pb )
     s.platform      = Gem::Platform::RUBY
 
-    s.add_dependency 'google-protobuf', '~> 3.13'
+    s.add_dependency 'google-protobuf', '~> ${settings.protobuf_short_version}'
```

Comment:
Just curious, why Ruby needs a shorter version number?

Reply:
Good question. I'm not sure, but the protobuf update doc explicitly called this out. Maybe @apolcyn knows?

Reply:
@gnossen which doc are you referring to?

If we list a protobuf patch release version here, then this gemspec will only permit the picked up google-protobuf dependency's patch version to change, i.e. the minor version will be pinned. But if we only list the minor version number, then the minor version number will be allowed to move.

Maybe `protobuf_short_version` should be called `protobuf_minor_version` ?

Reply:
I'm referring to go/grpc-third-party-protobuf-update-instructions:
> ruby: update "google-protobuf" version in templates/grpc.gemspec.template  to v3.X (WITHOUT the patch version - 3.x.y is NOT correct) and run tools/buildgen/generate_projects.sh. Check that the gem version is available  at https://rubygems.org/gems/google-protobuf

Technically, the short version includes both major and minor. I would expect a variable called `protobuf_minor_version` to (currently) be "13", not "3.13". How about `protobuf_major_minor_version`?  More verbose, but less confusing IMO.

Reply:
`protobuf_major_minor_version` SGTM